src/helpers/assets-manifest.js

```diff
@@ -1,18 +1,19 @@
 'use strict'
 
-const { execSync } = require('child_process')
-const fs = require('fs')
 const logger = require('@antora/logger')('assets-manifest-helper')
 
 module.exports = (assetPath) => {
-  let manifestPath
-  let manifest
-  try {
-    manifestPath = execSync('find ./build/site -name assets-manifest.json').toString().trim()
```

Comment:
I realized we can't read the UI bundle files at this point in the Antora build because the 'build' folder isn't created until after the page composer step in Anotra. The only way I see to access this assets-manifest file is using the uiCatalog in an Antora extension and somehow make that object available to this helper, thus the global context